src/apppermission.cpp

```diff
@@ -95,18 +95,19 @@ Q_INVOKABLE void AppPermission::flip(const QString& appID) {
     settingsHolder->addVpnDisabledApp(appID);
   }
 
-  int index = m_applist.keys().indexOf(appID);
+  int index = m_applist.indexOf(AppDescription(appID, "Test"));
```

Comment:
This is creating temporary `AppDescription` class so that `operator==()` can be used for the search of `m_applist` for the matching `appID`. The operator doesn't make use of the application name, but the class constructor still needs a value. I guess it would be cleaner to provide a default value in the constructor.